src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -299,6 +302,60 @@ class guild_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_GUILD_INFO_EXTRA_INFO, guild->GetInfo().c_str()); // Extra Information
         return true;
     }
+
+    static bool HandleGuildListCommand(ChatHandler* handler, char const* /*args*/)
+    {
+        auto const& guildStore = sGuildMgr->GetGuildStore();   // map<uint32, unique_trackable_ptr<Guild>>
+
+        handler->SendSysMessage("|cff00ff00=== Guild list ===|r");
+        handler->SendSysMessage("|cff00ffffID | Name | GM | Created | Members | Lv | Bank(g)|r");
+
+        constexpr uint64 COPPER_PER_GOLD = 10000ULL;           //  1 gold = 10 000 copper
+
+        for (auto const& pair : guildStore)
+        {
+            uint32 id   = pair.first;
+            Guild* g    = pair.second.get();
+            if (!g)
+                continue;
+
+            /* ----- GM name ----- */
+            std::string gmName;
+            if (!sCharacterCache->GetCharacterNameByGuid(g->GetLeaderGUID(), gmName))
+                gmName = "---";
+
+            /* ----- creation date (yyyy-mm-dd) ----- */
+            char dateBuf[11] = "---";
+            if (time_t created = g->GetCreatedDate())
+            {
+                tm tmBuf;
+    #ifdef _MSC_VER
+                localtime_s(&tmBuf, &created);
```

Comment:
Education purpose comment only (as this code is meant to be removed) - we provide `localtime_r` on windows in Util.h to avoid writing this kind of #ifdef blocks everywhere